app/src/main/java/hous/release/android/presentation/profile/edit/ProfileEditViewModel.kt

```diff
@@ -0,0 +1,71 @@
+package hous.release.android.presentation.profile.edit
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import hous.release.domain.repository.ProfileRepository
+import kotlinx.coroutines.launch
+import timber.log.Timber
+import javax.inject.Inject
+
+@HiltViewModel
+class ProfileEditViewModel @Inject constructor(
+    private val profileRepository: ProfileRepository
+) : ViewModel() {
+    val nickname = MutableLiveData<String>()
+
+    val birthday = MutableLiveData("")
+
+    private val _isBirthdayPublic = MutableLiveData(false)
```

Comment:
```suggestion
    private val isBirthdayPublic = MutableLiveData(false)
```  
여기서는 `backing`을 안해줘도 될 것 같습니다